cmux-tui/frontends/web/src/styles.css

```
@@ -168,7 +174,7 @@ button { color: inherit; touch-action: manipulation; }
 .render-underline-dotted { text-decoration-style: dotted; }
 .render-underline-dashed { text-decoration-style: dashed; }
 .render-run-blink { animation: render-text-blink 1s steps(1, end) infinite; }
-.render-cursor { position: absolute; z-index: 3; display: block; width: var(--render-cell-width); height: var(--render-cell-height); background: currentColor; opacity: .72; pointer-events: none; }
+.render-cursor { position: absolute; z-index: 4; display: block; width: var(--render-cell-width); height: var(--render-cell-height); background: currentColor; opacity: .72; pointer-events: none; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)